src/cryptonote_core/service_node_list.cpp

```diff
@@ -72,6 +72,12 @@ namespace service_nodes
 
   void service_node_list::init()
   {
+    if (m_blockchain.get_hard_fork_version() < 9)
```

Comment:
This does not compile, do you meant to pass `current_height` in?

Reply:
Fixed.